tsconfig.json

```diff
@@ -30,12 +30,13 @@
   "include": [
     "packages/global.d.ts",
     "packages/*/src",
+    "packages-private/*/src",
```

Comment:
It’s a good idea to create a separate tsconfig for `packages-private` instead of using the one in the root directory. This way, we’ll have more control over setting the target to `esnext`.

Reply:
<img width="673" alt="image" src="https://github.com/user-attachments/assets/88eabf9b-f119-4cb8-99c2-7dad5fcce270">